coalib/settings/ConfigurationGathering.py

```diff
@@ -397,19 +397,23 @@ def get_config_directory(section):
     return config if os.path.isdir(config) else os.path.dirname(config)
 
 
-def get_all_bears(log_printer=None, arg_parser=None, silent=True):
+def get_all_bears(log_printer=None,
+                  arg_parser=None,
+                  silent=True,
+                  bear_globs=tuple('**')):
```

Comment:
this is not what's intended, I believe you thought this creates a single-element tuple with `**` in it, right? ;)
However, `tuple()` collects any iterabel and puts it into a tuple. `string` is an iterable, it allows you to iterate over each char. So in this case you'll get:
`('*', '*')`

It has to be `('**',)` (note the comma)

Reply:
(and without `tuple`, so just `bear_globs=('**',)`)